### PR TITLE
Install the completions in the installation script

### PR DESCRIPTION
**Pull request by Duhemm**

### CODE REVIEW

**Comment by Duhemm** [comment]
I've tested the script with python2 and python3 :)

File: bin/install.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from os.path import expanduser, isdir, isfile, join
 from subprocess import CalledProcessError, check_call
 import sys
+import re
```
Comment (Duhemm) [inline bin/install.py]: This comes from the python stdlib, so no worries

**Comment by jvican** [comment]
:shipit:

File: bin/install.py [hunk 2]
```diff
@@ -53,11 +54,30 @@
...
+ZSH_COMPLETION_DIR = join(BLOOP_INSTALLATION_TARGET, "zsh")
+BASH_COMPLETION_DIR = join(BLOOP_INSTALLATION_TARGET, "bash")
+
+# If this is not a released version of Bloop, we need to extract the commit SHA
+# to know how to download the completion scripts.
+# If we can't get the SHA, just download from master.
+if CUSTOMIZED_SCRIPT:
+    COMPLETION_VERSION = BLOOP_VERSION
+else:
+    pattern = '(?:.+?)-([0-9a-f]{8})(?:\+\d{8}-\d{4})?'
```
Comment (Duhemm) [inline bin/install.py]: That's the same regex we use to do the same thing in the benchmarks

**Pull request merged**